### DISCUSSION TITLE
Failed to send transaction

### DISCUSSION THREAD

**Discussion by ArnaudBand** [Cyfrin/foundry-full-course-cu#2363] [Q&A]
## Account Abstraction

After finish to deploy my smart contract successfully, I'm still getting an error that my wallet doesn't have the funds. 

```bash
 shortMessage: 'insufficient funds for gas + value. balance: 0, fee: 37505150000000, value: 0'
}
```

Any support of how I can get funds because I have:
Testnet
zksync-sepolia = 0.1
usdc=10

**Comment by EngrPips** [comment]
Hello @ArnaudBand, What process did you run that caused you to get the below error?

```js
shortMessage: 'insufficient funds for gas + value. balance: 0, fee: 37505150000000, value: 0'
```

**Reply by ArnaudBand** [reply]
I deployed my ZksyncMinimalAccount contract and that's where I failed to send the transaction

**Reply by EngrPips** [reply]
Do you mean you ran your deployment script, and then you got that error?

**Reply by ArnaudBand** [reply]
I ran javascript scripts

**Reply by EngrPips** [reply]
Honestly, I didn't do the Javascript part since Patrick said it is optional but why not get more `zksync-sepolia` ether and retry, you can bridge `Ethereum-sepolia` ether to `zksync-sepolia` using https://portal.zksync.io/bridge/?network=sepolia

**Reply by ArnaudBand** [reply]
I have some zksync sepolia

**Reply by EngrPips** [reply]
You said you have `0.1` ether worth of `zksync-sepolia` eth, am I correct?

**Reply by ArnaudBand** [reply]
Yes

**Reply by EngrPips** [reply]
So my suggestion was that you get more `zksync-sepolia` eth and retry the process.
